src/tokens/TokenImport.tsx

```diff
@@ -204,8 +203,6 @@ export default function TokenImportScreen(_: Props) {
     const networkId = tokenDetails.networkId
     const tokenAddress = tokenDetails.address.toLowerCase()
 
-    // This should always be present, unless we support a network without tokens,
-    // or if we can't fetch the token list (e.g. network issues?), should we fail instead to be safe?
```

Comment:
This should be good for now, we are updating the network icon the fetching balance step anyways.